dotnetv3/S3/ListBucketsExample/ListBuckets.cs

```diff
@@ -1,24 +1,29 @@
 ﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX - License - Identifier: Apache - 2.0
+// SPDX-License-Identifier:  Apache-2.0
 
-using Amazon;
-using Amazon.S3;
-using Amazon.S3.Model;
-using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
-
-namespace ListBuckets
+/// <summary>
+/// This example uses the AWS SDK for .NET to list the Amazon Simple Storage
+/// Service (Amazon S3) buckets belonging to the default account. This code
+/// was written using AWS SDK for .NET v3.5 and .NET Core 5.0.
+/// </summary>
+namespace ListBucketsExample
 {
-    // This example uses the AWS SDK for .NET to list the Amazon Simple Storage
-    // Service (Amazon S3) buckets belonging to the default account. This code
-    // was written using AWS SDK for .NET v3.5 and .NET Core 5.0.
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Amazon.S3;
+    using Amazon.S3.Model;
+
     class ListBuckets
     {
         private static IAmazonS3 _s3Client;
 
         static async Task Main(string[] args)
         {
+            // The client uses the AWS Region of the default user
```

Comment:
add period

Reply:
Fixed. Also removed unused arguments to Main.